### PR TITLE
Merging down #74 into main

### PR DESCRIPTION
**Pull request by CheesyBoy123**
Motivation:
Merging down https://github.com/eclipse-vertx/vertx-openapi/issues/74

Conformance:

You should have signed the Eclipse Contributor Agreement as explained in https://github.com/eclipse/vert.x/blob/master/CONTRIBUTING.md
Please also make sure you adhere to the code style guidelines: https://github.com/vert-x3/wiki/wiki/Vert.x-code-style-guidelines

### CODE REVIEW

File: src/main/java/io/vertx/openapi/validation/SchemaValidationException.java [hunk 1]
```diff
@@ -51,6 +51,19 @@ public static SchemaValidationException createInvalidValueResponseBody(OutputUni
...
+                                                                        JsonSchemaValidationException cause) {
+    switch(outputUnit.getErrorType()) {
+      case MISSING_VALUE:
+        return createMissingRequiredParameter(parameter);
+      case INVALID_VALUE:
+      case NONE:
+      default:
+        return createInvalidValueParameter(parameter, outputUnit, cause);
+    }
+
```
Comment (pk-work) [inline src/main/java/io/vertx/openapi/validation/SchemaValidationException.java:64]: empty line

**Review by pk-work — CHANGES_REQUESTED** [review]

**Comment by pk-work** [comment]
How is this PR related to https://github.com/eclipse-vertx/vertx-openapi/issues/74?

File: src/main/java/io/vertx/openapi/validation/impl/RequestValidatorImpl.java [hunk 2]
```diff
@@ -44,8 +44,7 @@
 import static io.vertx.openapi.contract.Style.LABEL;
 import static io.vertx.openapi.contract.Style.MATRIX;
 import static io.vertx.openapi.contract.Style.SIMPLE;
-import static io.vertx.openapi.validation.SchemaValidationException.createInvalidValueParameter;
-import static io.vertx.openapi.validation.SchemaValidationException.createInvalidValueRequestBody;
+import static io.vertx.openapi.validation.SchemaValidationException.*;
```
Comment (pk-work) [inline src/main/java/io/vertx/openapi/validation/impl/RequestValidatorImpl.java:47]: star import

**Comment by CheesyBoy123** [comment]
> How is this PR related to #74?

I grabbed the wrong commit - it is fixed now.

**Review by pk-work — APPROVED** [review]

**Pull request merged**